backend/minors/EUS.yaml

```diff
@@ -0,0 +1,79 @@
+---
+type: Minor
+name: European Studies
+code: EUS
+degree:
+urls:
+- https://ecs.princeton.edu/minor
+description: The Minor in European Studies offers a comprehensive exploration of Europe, blending 
+  the Humanities and Social Sciences. Students gain a deep understanding of Europe’s past and present 
+  through diverse coursework and engaging extra-curricular activities. You’ll explore European history, 
+  literature, art, architecture, music, cinema, theater, politics, sociology, economy, and philosophy, 
+  covering various time periods and national contexts, and develop skills in critical thinking, cultural 
+  analysis, and interdisciplinary research.
+contacts:
+- type: Director
+  name: Efthymia Rentzou
+  email: erentzou@princeton.edu
+req_list:
+- name: Prerequisites
+  min_needed: 1
+  max_counted: 1
+  explanation: |-
+    One of the following core courses is required as a prerequisite: ECS 301 or EPS 302. Students who are 
+    unable to take either ECS 301 or EPS 302 may petition the directors for other courses that offer an 
+    introduction to the study of Europe from a similarly interdisciplinary point, such as HUM 216-217, 
+    HUM 218-219, or HIS 212/EPS 212.
+  course_list:
+  - ECS 301
+  - EPS 302
+- name: Electives
+  min_needed: 3
+  max_counted: 1
+  explanation: |-
+    In addition to the prerequisite course, students will be required to take four elective courses in ECS or EPS; 
+    at least one of these courses should be in ECS and one in EPS, core or cross-listed. Students who have taken a 
+    full year of the double-credit HUM 216-217 and HUM 218-219 (the Western Humanities Sequence) satisfy two of the 
+    elective courses. Students who have taken one semester of the double-credit HUM 216-217 or HUM 218-219 satisfy 
+    one of the elective courses.
+  req_list: 
+  - name: Four total courses
+    min_needed: 4
+    max_counted: 1
+    course_list:
+    - ECS ***
+    - EPS ***
+    - HUM 216
+    - HUM 218
+  - name: ECS Course
+    min_needed: 1
+    max_counted: 1
+    course_list:
+    - ECS ***
+  - name: EPS Course
+    min_needed: 1
+    max_counted: 1
+    course_list:
+    - EPS ***
```

Comment:
Let's reorganize the requirements so that it would be simpler to understand for a user. 
Keep "ECS Courses" and "EPS Courses" as the subreqs of "Electives", but make it so they have min_needed: 1 and max_counted: 3. This way, you can encode the fact that you need to take a course in the other department. Let me know if that makes sense.

Reply:
Gotcha I think that makes sense, I can re-arrange it in a bit. Do you mind explaining the function of max_counted? Think I might not be completely understanding/using it right.

Reply:
Oh I think that might not work if they take any HUM classes and that counts since they could have 3 EUS + 1 HUM

Reply:
`max_counted` is the maximum amount of requirements that can be fed to the parent category. Feel free to check out George's report for more details!

Reply:
I think you're right about the HUM classes... maybe we can keep it the way it is, and ensure we have `double_counting` as true

Reply:
should be updated now! thanks for the reminder, forgot about the report